controls/views.py

```diff
@@ -2092,6 +2092,7 @@ def save_smt(request):
                 statement.body = form_values['body']
                 statement.remarks = form_values['remarks']
                 statement.status = form_values['status']
+                statement.inheritance_id = form_values["inheritance_name"]
```

Comment:
Just want to confirm that the value of `form_values["inheritance_name"]` is *not* the name of the inheritance type, but actually the PK ...